### PR TITLE
#778 Deployments pagination

### LINKED ISSUE #778: [deployment] The API endpoint is paginated but the CLI does not let us use the pagination

### PR DESCRIPTION
**Pull request by SCedricThomas**
resolve #778

### CODE REVIEW

File: cmd/deployments.go [hunk 1]
```diff
@@ -41,13 +42,20 @@ var (
 		Name:     "deployments",
 		Category: "Deployment",
 		Usage:    "List app deployments",
-		Flags:    []cli.Flag{&appFlag},
+		Flags: []cli.Flag{
+			&appFlag,
+			&cli.IntFlag{Name: "page", Usage: "Page to display", Value: 1},
+			&cli.IntFlag{Name: "per-page", Usage: "Number of deployements to display", Value: 30},
+		},
```
Comment (curzolapierre) [inline cmd/deployments.go:49]: It missing the indentation, you should configure your IDE to automatically indent it on saving.

File: cmd/deployments.go [hunk 2]
```diff
@@ -41,13 +42,20 @@ var (
 		Name:     "deployments",
 		Category: "Deployment",
 		Usage:    "List app deployments",
-		Flags:    []cli.Flag{&appFlag},
+		Flags: []cli.Flag{
+			&appFlag,
+			&cli.IntFlag{Name: "page", Usage: "Page to display", Value: 1},
+			&cli.IntFlag{Name: "per-page", Usage: "Number of deployements to display", Value: 30},
```
Comment (curzolapierre) [inline cmd/deployments.go:48]:
```suggestion
			&cli.IntFlag{Name: "per-page", Usage: "Number of deployments to display", Value: 30},
```

Comment (curzolapierre) [inline cmd/deployments.go:48]: Out of curiosity why do use 30 per page by default as the default value per page value is 20 on the API?

Comment (curzolapierre) [reply cmd/deployments.go:48]: Think to add a spell checker on your IDE to avoid this kind of mistakes.

**Review by curzolapierre — CHANGES_REQUESTED** [review]
Could you please provide an example of the output by copy pasting it in the description of this PR (or provide a screenshot)?

Just small nitpicking comments, overall seems good

File: cmd/deployments.go (hunk 2 above)
Comment (SCedricThomas) [reply cmd/deployments.go:48]: Because the only other feature using pagination was configured with 30 by default

**Comment by curzolapierre** [comment]
You didn't take in account my previous comment related to the description:
> Could you please provide an example of the output by copy pasting it in the description of this PR (or provide a screenshot)?

**Review by curzolapierre — COMMENTED** [review]
You didn't take in account my previous comment related to the description:
> Could you please provide an example of the output by copy pasting it in the description of this PR (or provide a screenshot)?

**Comment by SCedricThomas** [comment]
Oops ^^'
Take a look !  
Screenshot from 2022-09-07 15-17-46

**Review by curzolapierre — APPROVED** [review]
LGTM

**Pull request merged**